Sources/tart/VNCWrapper.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import Dynamic
+import Virtualization
+
+class VNCWrapper {
+    private let vnc: Dynamic
+    private let vncCredentialsURL: URL
+
+    init(virtualMachine: VZVirtualMachine, vncCredentialsURL: URL) async throws {
+        let password = UUID().uuidString
```

Comment:
Maybe just use `admin` password? Or something like `password101` 😅

Reply:
Can you explain your use-case a little better? Because the `tart vnc --open` relieves the paint of typing or copy-paste'ing that by hand.

Reply:
`tart vnc` is relying on the credentials being stored aka VM started with `tart run --vnc`.

But my comment was mostly for the UX since we'll print the `vnc://...` and it will be nice IMO to not have long UUID there.

Reply:
> But my comment was mostly for the UX since we'll print the `vnc://...` and it will be nice IMO to not have long UUID there.

Would random human-readable passphrase work?

Reply:
SGTM

Reply:
See https://github.com/cirruslabs/tart/pull/126/commits/86563410962f2b1040a8db93ade15d7344004dee.